apps/src/templates/sectionProgressV2/IconKey.jsx

```diff
@@ -34,17 +41,33 @@ export default function IconKey({isViewingValidatedLevel, expandedLessonIds}) {
       aria-expanded={isOpen}
       aria-label={i18n.iconKey()}
     >
-      <div
-        onClick={clickListener}
-        className={styles.iconKeyTitle}
-        data-testid="expandable-container"
-      >
-        <Heading6>
-          <FontAwesome className={styles.iconKeyCaret} icon={caret(isOpen)} />
-          {i18n.iconKey()}
-        </Heading6>
+      <div className={styles.iconKeyHeader}>
+        <div
```

Comment:
Also though... I needed an extra `div` here to prevent having a user click on the `Link` and unintentionally opening up the expandable `div`.  This was the solution I came up with, but am open to others.  I tried making an `expandable Heading6` but that didn't work...